rtl_tests.c

```diff
@@ -211,8 +211,80 @@ void test_RtlAssert(){
     /* FIXME: This is a stub! implement this function! */
 }
 
+// FIXME: RtlCaptureContext also sets Ebp, Eip, and Esp in the context record.
+// However, without doing some math there isn't a good way to know what Ebp, Esp, and Eip were
+// set to in the caller of RtlCaptureContext. And I'm too lazy to figure out that code right now.
 void test_RtlCaptureContext(){
-    /* FIXME: This is a stub! implement this function! */
+    const char* func_num = "0x0109";
+    const char* func_name = "RtlCaptureContext";
+    BOOL tests_passed = 1;
+    print_test_header(func_num, func_name);
+
+    CONTEXT result_context;
```

Comment:
Shouldn't `ContextFlags` be set, or be checked?

Is it an input or output to the function you are testing?

It would probably affect the registers you are allowed to test.

Reply:
In my kernel dump of RtlCaptureContext, ContextFlags are not touched. CONTEXT addresses 0x21C, 0x218, 0x214, 0x210, 0x20C, 0x208, 0x228, 0x234, 0x22C, 0x224, 0x220, and 0x230 are set in the function which line up with the addresses for the flags and general purpose registers.

Reply:
Probably worth adding a comment here, or even in nxdk.